modules/ensemble/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ description: Ensemble Runtime
 # This version is used _only_ for the Runner app, which is used if you just do
 # a `flutter run` or a `flutter make-host-app-editable`. It has no impact
 # on any other native host app that you embed your Flutter project into.
-version: 1.0.0+5
+version: 1.0.1
```

Comment:
Today this auto-incment (in iOS). Do we have to reember to change this manually every time we push to Testflight?

Reply:
No will be auto-increment only.